packages/@aws-cdk/aws-rds/test/serverless-cluster.test.ts

```diff
@@ -3,9 +3,12 @@ import * as ec2 from '@aws-cdk/aws-ec2';
 import * as iam from '@aws-cdk/aws-iam';
 import * as kms from '@aws-cdk/aws-kms';
 import * as cdk from '@aws-cdk/core';
+import * as cxapi from '@aws-cdk/cx-api';
 import { nodeunitShim, Test } from 'nodeunit-shim';
 import { AuroraPostgresEngineVersion, ServerlessCluster, DatabaseClusterEngine, ParameterGroup, AuroraCapacityUnit, DatabaseSecret } from '../lib';
 
+const flags = { [cxapi.RDS_CLUSTER_IDENTIFIER_TO_LOWERCASE]: true };
```

Comment:
I don't like this global variable (makes it really difficult to understand what's inside such a generally named variable).

This variable is used exactly once, let's inline it there, and remove it.